### PR TITLE
Add mangrove tables (1.1.2), make azalea and jacaranda mineable.

### PR DESCRIPTION
**Pull request by sleepingdragoninn**
Changes:
 - 1.1.2 changes - mangrove block loot tables and mineability
   - Fix duplicate listings for mangrove in mineable tags
 - Adds jacaranda and azalea blocks to the mineable/axe tag
 - Adds jacaranda and azalea post caps / lanterns (uses amethyst lantern textures/recipes)

### CODE REVIEW

**Comment by sleepingdragoninn** [comment]
Tested: 

https://github.com/DebuggyTeam/architecture-extensions/assets/55819817/eca7d49a-88c5-410a-9b99-77286522b0f3

**Comment by sleepingdragoninn** [comment]
& with using amethyst instead of soul (oops!)

Amethyst post lamps won't have the cool 32-block spawn cancel effect, but maybe in 2.0 aurora's deco compat we can ask for a tag ^^

https://github.com/DebuggyTeam/architecture-extensions/assets/55819817/21746be9-bf2b-42e4-a6e1-2542d8a9b9c7

File: src/main/java/io/github/woodiertexas/architecture_extensions/compat/ArchExAurorasDecoCompat.java [hunk 1]
```diff
@@ -1,9 +1,6 @@
 package io.github.woodiertexas.architecture_extensions.compat;
 
-import io.github.woodiertexas.architecture_extensions.blocks.BeamBlock;
-import io.github.woodiertexas.architecture_extensions.blocks.FencePostBlock;
-import io.github.woodiertexas.architecture_extensions.blocks.JoistBlock;
-import io.github.woodiertexas.architecture_extensions.blocks.MoldingBlock;
+import io.github.woodiertexas.architecture_extensions.blocks.*;
```
Comment (falkreon) [inline src/main/java/io/github/woodiertexas/architecture_extensions/compat/ArchExAurorasDecoCompat.java]: I don't really like star imports.

**Review by falkreon — APPROVED** [review]
I have one small nitpick, but looks really good

File: src/main/java/io/github/woodiertexas/architecture_extensions/compat/ArchExAurorasDecoCompat.java (hunk 1 above)
Comment (sleepingdragoninn) [reply src/main/java/io/github/woodiertexas/architecture_extensions/compat/ArchExAurorasDecoCompat.java]: sorted in c882b30!

**Review by falkreon — APPROVED** [review]
:white_check_mark:

File: src/main/resources/data/minecraft/tags/blocks/mineable/axe.json [hunk 2]
```diff
@@ -56,11 +64,15 @@
...
 		"architecture_extensions:crimson_post_lantern",
 		"architecture_extensions:warped_post_cap",
-		"architecture_extensions:warped_post_lantern"
+		"architecture_extensions:warped_post_lantern",
+		"architecture_extensions:mangrove_post_cap",
+    "architecture_extensions:mangrove_post_lantern",
+    "architecture_extensions:jacaranda_post_cap",
+    "architecture_extensions:jacaranda_post_lantern",
+    "architecture_extensions:azalea_post_cap",
+    "architecture_extensions:azalea_post_lantern"
```
Comment (Ampflower) [inline src/main/resources/data/minecraft/tags/blocks/mineable/axe.json]: Bad indentation moment

File: src/main/resources/data/minecraft/tags/blocks/needs_stone_tool.json [hunk 3]
```diff
@@ -146,6 +158,12 @@
...
 		"architecture_extensions:crimson_post_lantern",
 		"architecture_extensions:warped_post_cap",
-		"architecture_extensions:warped_post_lantern"
+		"architecture_extensions:warped_post_lantern",
+    "architecture_extensions:mangrove_post_cap",
+    "architecture_extensions:mangrove_post_lantern",
+    "architecture_extensions:jacaranda_post_cap",
+    "architecture_extensions:jacaranda_post_lantern",
+    "architecture_extensions:azalea_post_cap",
+    "architecture_extensions:azalea_post_lantern"
```
Comment (Ampflower) [inline src/main/resources/data/minecraft/tags/blocks/needs_stone_tool.json]: Seems to be the same here

**Review by Ampflower — COMMENTED** [review]
It's mainly just styling issues that I can see; tho the files evidently didn't respect .editorconfig.

Otherwise looks good to me.

File: src/main/resources/data/minecraft/tags/blocks/mineable/axe.json (hunk 2 above)
Comment (sleepingdragoninn) [reply src/main/resources/data/minecraft/tags/blocks/mineable/axe.json]: Sorted in a4af478

File: src/main/resources/data/minecraft/tags/blocks/needs_stone_tool.json (hunk 3 above)
Comment (sleepingdragoninn) [reply src/main/resources/data/minecraft/tags/blocks/needs_stone_tool.json]: Sorted in a4af478

**Review by Ampflower — APPROVED** [review]

**Pull request merged**